templates/todo/projects/nodejs-mongo-aks/.repo/bicep/infra/main.bicep

```diff
@@ -92,17 +93,40 @@ module monitoring '../../../../../../common/infra/bicep/core/monitor/monitoring.
   }
 }
 
+// Manage configuration in Azure App Configuration
+module azappconfig '../../../../../../common/infra/bicep/core/config/configstore.bicep' = {
+  name: 'azappconfig'
+  scope: rg
+  params: {
+    name: !empty(configStoreName) ? configStoreName : '${abbrs.appConfigurationStores}${resourceToken}'
+    location: location
+    tags: tags
+    keyValueNames: [
+      'AZURE_AKS_IDENTITY_CLIENT_ID$todo-api'
+      'AZURE_KEY_VAULT_ENDPOINT$todo-api'
+      'APPLICATIONINSIGHTS_CONNECTION_STRING$todo-api'
+      'REACT_APP_APPLICATIONINSIGHTS_CONNECTION_STRING$todo-web'
```

Comment:
Can you explain the need for $todo-....

Reply:
This is the format defined by Azure App Configuration which is "key$label" format, we use the label to differentiate the configurations for different services.